pkg/configuration/configuration.go

```diff
@@ -143,6 +143,12 @@ const (
 	// varVerificationCodeExpiresInMin is used to set the amount of time (in minutes) that a verification code is active for before expiring
 	varVerificationCodeExpiresInMin     = "verification.code_expires_in_min"
 	DefaultVerificationCodeExpiresInMin = 5
+
+	// varWoopraDomain contains the woopra domain
+	varWoopraDomain = "woopra_domain"
+
+	// varSegmentWriteKey contains the segment write key
+	varSegmentWriteKey = "segment_write_key"
```

Comment:
ok, now you use the snake. btw, wouldn't it be good to use the prefix `woopra.` to express what it is used for?

Reply:
in addition, you are missing tests for these values

Reply:
this is consistent with what we have in this file - everything else uses snake here, see here for example: https://github.com/codeready-toolchain/registration-service/blob/master/pkg/configuration/configuration.go#L65

Reply:
cool, but you still forgot to add the test for it.

Reply:
sorry! wasn't meaning to ignore the test comment! fixed here: 7850984b18856fa004145dbfb3f9de3afc6fbfdd